src/python/WMComponent/DBS3Buffer/Oracle/Create.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, logger = None, dbi = None, params = None):
                  subscribed         INTEGER      DEFAULT 0,
                  phedex_group       VARCHAR(100),
                  delete_blocks      INTEGER,
+                 dataset_lifetime   INTEGER     DEFAULT 46656000,
```

Comment:
You updated this default for MySQL, but you missed it here for Oracle.

Reply:
Yes, sorry for that mistake Alan.